lib/widgets/dialog.dart

```diff
@@ -21,11 +21,11 @@ Widget _dialogActionText(String text) {
 ///
 /// See also:
 ///  * [showDialog], whose return value this class is intended to wrap.
-class DialogStatus {
+class DialogStatus<T> {
```

Comment:
It might be helpful to document the meaning of the type parameter. Specially, from `showDialog`, this part seems relevant:

```dart
/// If the application has multiple [Navigator] objects, it may be necessary to
/// call `Navigator.of(context, rootNavigator: true).pop(result)` to close the
/// dialog rather than just `Navigator.pop(context, result)`.
///
/// Returns a [Future] that resolves to the value (if any) that was passed to
/// [Navigator.pop] when the dialog was closed.
```

We just need to adapt it to fit our wrapper class.